doc/user/content/release-notes.md

```diff
@@ -46,6 +46,11 @@ Use relative links (/path/to/doc), not absolute links
 Wrap your release notes at the 80 character mark.
 {{< /comment >}}
 
+{{% version-header v0.8.0 %}}
```

Comment:
I moved this to 0.8.0 during rebase of the original commit. Let me know whether it should have been 0.7.4 instead (or something entirely different).